app/containers/ItunesAppContainer/PlayingSongBar/tests/index.test.js

```diff
@@ -0,0 +1,22 @@
+/**
+ *
+ * Tests for PlayingSongBar
+ *
+ */
+
+import React from 'react';
+// import { fireEvent } from '@testing-library/dom'
+import { renderWithIntl } from '@utils/testUtils';
+import PlayingSongBar from '../index';
+
+describe('<PlayingSongBar />', () => {
```

Comment:
write more test cases for how the component would behave with the props that are passed  in